code/controllers/subsystem/statpanel.dm

```diff
@@ -29,123 +39,162 @@ SUBSYSTEM_DEF(statpanels)
 		encoded_global_data = url_encode(json_encode(global_data))
 		src.currentrun = GLOB.clients.Copy()
 		mc_data_encoded = null
+
 	var/list/currentrun = src.currentrun
 	while(length(currentrun))
 		var/client/target = currentrun[length(currentrun)]
 		currentrun.len--
+
 		if(!target.statbrowser_ready)
 			continue
-		if(target.stat_tab == "Status")
-			var/ping_str = url_encode("Ping: [round(target.lastping, 1)]ms (Average: [round(target.avgping, 1)]ms)")
-			var/other_str = url_encode(json_encode(target.mob.get_status_tab_items()))
-			target << output("[encoded_global_data];[ping_str];[other_str]", "statbrowser:update")
+
+		if(target.stat_tab == "Status" && num_fires % status_wait == 0)
+			set_status_tab(target)
+
 		if(!target.holder)
 			target << output("", "statbrowser:remove_admin_tabs")
 		else
 			target << output("[!!(target.prefs.toggles & SPLIT_ADMIN_TABS)]", "statbrowser:update_split_admin_tabs")
+
 			if(!("MC" in target.panel_tabs) || !("Tickets" in target.panel_tabs))
 				target << output("[url_encode(target.holder.href_token)]", "statbrowser:add_admin_tabs")
-			if(target.stat_tab == "MC")
-				var/turf/eye_turf = get_turf(target.eye)
-				var/coord_entry = url_encode(COORD(eye_turf))
-				if(!mc_data_encoded)
-					generate_mc_data()
-				target << output("[mc_data_encoded];[coord_entry]", "statbrowser:update_mc")
-			if(target.stat_tab == "Tickets")
-				var/list/ahelp_tickets = GLOB.ahelp_tickets.stat_entry()
-				target << output("[url_encode(json_encode(ahelp_tickets))];", "statbrowser:update_tickets")
-				var/datum/interview_manager/m = GLOB.interviews
-
-				// get open interview count
-				var/dc = 0
-				for (var/ckey in m.open_interviews)
-					var/datum/interview/I = m.open_interviews[ckey]
-					if (I && !I.owner)
-						dc++
-				var/stat_string = "([m.open_interviews.len - dc] online / [dc] disconnected)"
-
-				// Prepare each queued interview
-				var/list/queued = list()
-				for (var/datum/interview/I in m.interview_queue)
-					queued += list(list(
-						"ref" = REF(I),
-						"status" = "\[[I.pos_in_queue]\]: [I.owner_ckey][!I.owner ? " (DC)": ""] \[INT-[I.id]\]"
-					))
-
-				var/list/data = list(
-					"status" = list(
-						"Active:" = "[m.open_interviews.len] [stat_string]",
-						"Queued:" = "[m.interview_queue.len]",
-						"Closed:" = "[m.closed_interviews.len]"),
-					"interviews" = queued
-				)
-
-				// Push update
-				target << output("[url_encode(json_encode(data))];", "statbrowser:update_interviews")
+
+			if(target.stat_tab == "MC" && ((num_fires % mc_wait == 0) || target?.prefs.read_preference(/datum/preference/toggle/fast_mc_refresh)))
```

Comment:
please start a soyouhaveplusdevontheserver.md document somewhere that lists out this pref, and any other useful information we find relevant, to prevent such thing sgetting lost to history

Reply:
I agree with this

Reply:
alright i added a md file on the MC tab and documented this pref